ComputerInterface/BaseGameInterface.cs

```diff
@@ -87,6 +87,37 @@ public static ETurnMode GetTurnMode()
             return (ETurnMode) Enum.Parse(typeof(ETurnMode), turnMode);
         }
 
+        public static void SetInstrumentVolume(int value)
+        {
+            PlayerPrefs.SetFloat("instrumentVolume", (float)value / 50f);
+            PlayerPrefs.Save();
+        }
+
+        public static float GetInstrumentVolume()
+        {
+            float instVolume = PlayerPrefs.GetFloat("instrumentVolume", 0.1f);
+            return (instVolume);
```

Comment:
This can just be
```suggestion
            return PlayerPrefs.GetFloat("instrumentVolume", 0.1f);
```

Reply:
ughhh so much stuff i couldve optimized that i missed 😭